Formula/libtecla.rb

```diff
@@ -14,8 +14,36 @@ class Libtecla < Formula
   end
 
   def install
-    ENV.deparallelize
-    system "./configure", "--prefix=#{prefix}", "--mandir=#{man}"
-    system "make", "install"
+    ENV.deparallelize do
+      system "./configure", "prefix=#{prefix}", "mandir=#{man}"
```

Comment:
This is a traditional `configure`, the options need to be of the form `--prefix`